conf/gunicorn/dev.conf.py

```diff
@@ -1,6 +0,0 @@
-# Configuration settings for Gunicorn in dev
-
-workers = 4
-reload = True
-bind = "0.0.0.0:9099"
-timeout = 0
```

Comment:
Checkmate had its dev Gunicorn configuration file in its own silly location, moved it to `conf/gunicorn-dev.conf.py` the same as in other apps. Also removed the silly comment from the top, changed `workers` from `4` to `2`, and added `reload_extra_files = "checkmate/templates"` but otherwise this config should be the same.